Ginger/Ginger/UserControlsLib/UCTreeView/UCTreeView.xaml.cs

```diff
@@ -40,7 +40,8 @@ public partial class UCTreeView : UserControl, IDragDrop
         public event EventHandler ItemDoubleClick;
         public event EventHandler ItemDropped;
         public delegate void ItemDroppedEventHandler(DragInfo DI);
-        public bool TreeItemDoubleClicked = false;        
+        public bool TreeItemDoubleClicked = false;
+        public bool TreeChildFolderOnly = false;
 
```

Comment:
i see you added the TreeChildFolderOnly but didn't used it, you should use it inside SetTreeNodeItemChilds() so it will be generic without touching the original folder treeItems - but it's your call to decide what is better